### PR TITLE
Iteration 3

### PR DESCRIPTION
**Pull request by laurenlucero**
# Pull Request

### If it is a new feature, describe the feature and the iteration of the project spec it addresses. Note any questions or struggles as well.
Guest user can:
-select a date to book a room
-see a list of room details for rooms available on that date
-select a room for booking with a form, want to add functionality to click book room button
-filter the list of available rooms by room type
-get a message that no rooms are available for date/type selected but this has not been tested
-data posts to endpoint, checked manually, to be tested
Manager can also book a room for a user.

### Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Checklist:
- [ ] My code follows the Turing style guidelines - I am not sure if the formatting of my event listeners meets guidelines
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### Files Modified:
	modified:   README.md
	modified:   src/domUpdates.js
	modified:   src/index.html
	modified:   src/index.js

### CODE REVIEW

**Comment by laurenlucero** [comment]
@Karlfunhouse @hannahhch @thatpamiam

File: src/Hotel.js [hunk 1]
```diff
@@ -36,6 +36,25 @@ class Hotel {
     });
     return roomsAvailable;
   }
+
+  bookRoom(userID, date, roomNumber) {
```
Comment (Karlfunhouse) [inline src/Hotel.js:40]: Nice work getting your POST dialed in!

File: src/index.html [hunk 2]
```diff
@@ -58,61 +60,120 @@ <h2>
...
+              type="date"
+              name="guestID"
+              id="date-input"
+              class="form-field"
+              value="yyyy-MM-dd"
+              min="2020-01-01"
+              max="2022-12-31"
+              required
+            />
+            <label for="room-input">Room Number</label>
```
Comment (Karlfunhouse) [inline src/index.html:161]: Are you going to have the user manually input the room number?

**Comment by Karlfunhouse** [comment]
@thatpamiam @hannahhch Another Teamwork PR

File: src/index.html (hunk 2 above)
Comment (laurenlucero) [reply src/index.html:161]: As of now, yes, future iteration they could also click a button to book specific room

**Pull request merged**